tests/cards.js

```diff
@@ -3,16 +3,25 @@ const should = require('should');
 const cards = require('../src/cards');
 const cardsData = require('../data/cards');
 
-describe('cards test', () => {
-    it('should return a card template', () => {
+describe('Cards tests', () => {
```

Comment:
change this to `decribe('Cards', () => {...`
These strings end up generating the names of each test.
As of right now, it reads "Cards tests should return all the cards" which doesn't make sense.

Also, some people (I saw this in the ruby community) are changing the narrative from "should" to "expectations" (You can read more about it [here](http://betterspecs.org/#expect)). 
If I recall correctly the argument is that if you say "A should do B" it could be implied that you have no idea what it is going to do, on the other hand, if you say "A is expected to do B" you are not leaving room for ambiguity. 

PS: I'm not requesting you to change all the should narrative, I'm just sharing that there are different points of view.